show/plugins/barefoot.py

```diff
@@ -22,24 +22,32 @@ def profile():
     hwsku_dir = device_info.get_path_to_hwsku_dir()
     with open(hwsku_dir + '/switch-tna-sai.conf') as file:
         chip_family = json.load(file)['chip_list'][0]['chip_family'].lower()
-    
+
     # Print current profile
     click.echo('Current profile: ', nl=False)
     subprocess.run('docker exec -it syncd readlink /opt/bfn/install | sed '
-        r's/install_\\\(.\*\\\)_profile/\\1/', check=True, shell=True)
-    
-    # Exclude current and unsupported profiles
+        r's/install_\\\(.\*\\\)_profile/\\1/'
+        r' | sed s/install_\\\(.\*\\\)_tofino\\\(.\*\\\)/\\1/', check=True, shell=True)
```

Comment:
The `\(...\)` is a capture group. It captures the part of the regular expression. There can be multiple capture groups that then can be referenced with `\1`, `\2`, etc.
In the regular expression above the second capture group is not used, so it can be omitted, like so:
```
        r' | sed s/install_\\\(.\*\\\)_tofino.\*/\\1/', check=True, shell=True)
```